android/src/main/res/layout/custom_notification_incoming.xml

```diff
@@ -1,65 +1,69 @@
-    android:layout_width="wrap_content"
-    android:layout_height="wrap_content"
-    android:layout_marginBottom="@dimen/activity_horizontal_margin"
-    android:layout_marginLeft="@dimen/activity_horizontal_margin"
-    android:layout_marginRight="@dimen/activity_horizontal_margin"
-    android:textColor="@color/colorContent"
-    android:visibility="visible" />
-
   <LinearLayout
+    android:orientation="vertical"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:layout_marginBottom="@dimen/activity_horizontal_margin"
-    android:orientation="horizontal">
+    android:layout_height="140dp">
 
-    <Button
-      android:id="@+id/button_decline"
-      android:layout_width="match_parent"
+    <TextView
+      android:id="@+id/notif_title"
       android:layout_height="wrap_content"
+      android:layout_width="wrap_content"
+      android:layout_marginTop="@dimen/activity_horizontal_margin"
       android:layout_marginRight="@dimen/activity_horizontal_margin"
       android:layout_marginLeft="@dimen/activity_horizontal_margin"
-      android:layout_weight="0.5"
-      android:radius="10dp"
+      android:textSize="20dp"
       android:textStyle="bold"
-      android:textSize="16dp"
-      android:textColor="@color/colorButtonText"
-      android:background="@drawable/button_background_red"
-      android:text="@string/decline" />
+      android:textColor="@color/colorContentTitle"
+      android:visibility="visible"/>
 
-    <Button
-      android:id="@+id/button_answer"
-      android:layout_width="match_parent"
+    <TextView
+      android:id="@+id/notif_content"
+      android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      android:layout_marginRight="@dimen/activity_horizontal_margin"
+      android:layout_marginBottom="@dimen/activity_horizontal_margin"
       android:layout_marginLeft="@dimen/activity_horizontal_margin"
-      android:layout_weight="0.5"
-      android:textStyle="bold"
-      android:textSize="16dp"
-      android:textColor="@color/colorButtonText"
-      android:background="@drawable/button_background_green"
-      android:text="@string/answer" />
+      android:layout_marginRight="@dimen/activity_horizontal_margin"
+      android:textColor="@color/colorContent"
+      android:visibility="visible" />
 
-  </LinearLayout>
+    <LinearLayout
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:layout_marginBottom="@dimen/activity_horizontal_margin"
+      android:orientation="horizontal">
 
-</LinearLayout>
+      <Button
+        android:id="@+id/button_decline"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginRight="@dimen/activity_horizontal_margin"
+        android:layout_marginLeft="@dimen/activity_horizontal_margin"
+        android:layout_weight="0.5"
+        android:radius="10dp"
```

Comment:
Noticed that this button has a radius but the answer button does not. Is that intentional?

Reply:
No idea, I didn't modify that code, just moved it into a FrameLayout